.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -12,6 +12,7 @@ on:
       - completed
 jobs:
   build:
+    if: ${{ github.repository_owner == 'alteryx' }}
```

Comment:
The outer wrapper `${{ }}` is not necessary

Reply:
fixed